internal/run/run.go

```diff
@@ -454,6 +457,57 @@ func (w *Workflow) runSource(ctx context.Context, parentStep *WorkflowStep, id s
 		}
 	}
 
+	if os.Getenv("SPEAKEASY_PUBLISH_ARTIFACT") == "1" {
```

Comment:
nit: but why 1 here. Will we have multiple values for this env variable.

Reply:
Answered privately